### ISSUE TITLE
pipes are not redirected back in 3.1.0

### ISSUE THREAD

**Issue by scturtle** [minrk/wurlitzer#88]
```python
import wurlitzer
with open("log", "w") as f, wurlitzer.pipes(f):
        print("foo")
print("bar")
```

In 3.0.3, it will print `bar`. But in 3.1.0 it will print nothing and `bar` is also in `log`. Tested on MacOS and Linux.

**Comment by scturtle** [comment]
```python
import os
import wurlitzer
cpid = os.fork()
if cpid == 0:
    with open("log", "w") as f, wurlitzer.pipes(f):
            print("foo")
else:
    os.waitpid(cpid, 0)
    print("bar")
```

For this code, 3.1.0 seems to be right. But 3.0.3 puts nothing in `log` file.

**Comment by minrk** [comment]
Thanks! Tested and fixed by #89

**Comment by minrk** [comment]
released 3.1.1 with the fix
